data/meson.build

```diff
@@ -14,7 +14,7 @@ install_data(['org.freedesktop.fwupd.xml'],
 )
 
 install_data(['90-fwupd-devices.rules'],
-  install_dir : join_paths(get_option('prefix'), 'lib', 'udev', 'rules.d')
+  install_dir : '/lib/udev/rules.d'
```

Comment:
Is it usual to have a leading slash? I've not seen that in other meson projects.